### DISCUSSION TITLE
Drawer组件文档里出现 5.19.0 的 api, 但是5.19.0 貌似还未发布？

### DISCUSSION THREAD

**Discussion by luichooy** [ant-design/ant-design#49340] [Q&A]
![image](https://github.com/ant-design/ant-design/assets/26423033/ee02919c-6fc8-46d1-9655-fce5e44f6899)

**Comment by luichooy** [comment]
另外想问下这个 api 是用来支持类似 Modal 组件中的自定义拖拽功能吗？目前对 Drawer 组件的 resizable 功能有需求。目前自己结合react-resizable实现，但是会有卡顿和拖拽不跟鼠标的情况，请问官方有指导方案吗

**Reply by zombieJ** [reply]
drag、resize、motion 这种高频交互其实走 React 生命周期性能会比较差，业界有一些方案是直接设置 dom 元素来提高性能。

**Comment by yoyo837** [comment]
笔误 https://github.com/ant-design/ant-design/pull/49208

**Comment by zombieJ** [comment]
嗯，文档错了。已经有 PR 修了：https://github.com/ant-design/ant-design/pull/49208

发版本时会自动更新文档~

**Reply by crazyair** [reply]
额，我的错
